contracts/pm/mixins/MixinReserve.sol

```diff
@@ -133,92 +103,35 @@ contract MixinReserve is MContractRegistry, MReserve {
         internal
         returns (uint256)
     {
-        ReserveManager storage manager = reserveManagers[_reserveHolder];
-        Reserve storage reserve = manager.reserve;
-
-        // If reserve is not frozen then freeze it
-        if (reserveState(_reserveHolder) == ReserveState.NotFrozen) {
-            uint256 freezeRound = roundsManager().currentRound();
-            // TODO: make sure bondingManager.getTranscoderPoolSize()
-            // returns the locked in # registered for the freeze round
-            uint256 recipientsInFreezeRound = bondingManager().getTranscoderPoolSize();
-
-            reserve.freezeRound = freezeRound;
-            reserve.recipientsInFreezeRound = recipientsInFreezeRound;
-
-            emit ReserveFrozen(
-                _reserveHolder,
-                _claimant,
-                freezeRound,
-                recipientsInFreezeRound
-            );
-        }
+        Reserve storage reserve = reserves[_reserveHolder];
```

Comment:
I think we can move this inside of the `if claimAmount > 0` block

Reply:
Fixed in 1723500